tests/test_input_damages.py

```diff
@@ -0,0 +1,1287 @@
+        },
+    )
+
+    ds_out_actual = concatenate_labor_damages(
+        input_path=os.path.join(tmp_path, "labor_in"),
+        save_path=tmp_path,
+        ec_cls=econvars_fixture,
+    )
+
+    if out_format == "return":
+        xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+    elif out_format == "save":
+        xr.testing.assert_equal(
+            ds_out_expected,
+            xr.open_dataset(os.path.join(tmp_path, "rebased_wage-levels_batch9.nc4")),
+        )
+
+
+def test_error_concatenate_labor_damages(
+    caplog,
+    econvars_fixture,
+    tmp_path,
+    labor_in_val_fixture,
+    labor_in_histclim_fixture,
+):
+    """
+    Test that concatenate_labor_damages returns an exception when any batch cannot be processed succesfully
+    """
+    os.makedirs(
+        os.path.join(tmp_path, "labor_in", "batch6", "rcp45", "CCSM4", "high", "SSP3")
+    )
+    concatenate_labor_damages(
+        input_path=os.path.join(tmp_path, "labor_in"),
+        save_path=tmp_path,
+        ec_cls=econvars_fixture,
+    )
+    assert "Error in batchbatch6" in caplog.text
+
+    shutil.rmtree(
+        os.path.join(tmp_path, "labor_in", "batch6", "rcp45", "CCSM4", "high", "SSP3")
+    )
+
+
+def test_calculate_labor_batch_damages(
+    tmp_path,
+    econvars_fixture,
+    labor_in_val_fixture,
+    labor_in_histclim_fixture,
+):
+    """
+    Test that calculate_labor_batch_damages correctly concatenates damages across batches when single batch is passed
+    """
+    ds_out_expected = xr.Dataset(
+        {
+            "histclim_rebased": (
+                ["ssp", "rcp", "model", "gcm", "batch", "year", "region"],
+                np.float32(np.full((1, 2, 2, 2, 1, 2, 2), 2 * -1 * 1.273526)),
+            ),
+            "delta_rebased": (
+                ["ssp", "rcp", "model", "gcm", "batch", "year", "region"],
+                np.float32(np.full((1, 2, 2, 2, 1, 2, 2), 0)),
+            ),
+        },
+        coords={
+            "year": (["year"], [2010, 2099]),
+            "region": (["region"], ["ZWE.test_region", "USA.test_region"]),
+            "ssp": (["ssp"], ["SSP3"]),
+            "rcp": (["rcp"], ["rcp45", "rcp85"]),
+            "gcm": (["gcm"], ["ACCESS1-0", "GFDL-CM3"]),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "batch": (["batch"], ["batch6"]),
+        },
+    )
+
+    calculate_labor_batch_damages(
+        batch=6,
+        ec=econvars_fixture,
+        input_path=os.path.join(tmp_path, "labor_in"),
+        save_path=tmp_path,
+    )
+
+    ds_out_actual = xr.open_zarr(
+        os.path.join(tmp_path, "rebased_wage-levels_batch6.zarr")
+    )
+
+    xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+
+def test_calculate_labor_damages(
+    tmp_path,
+    labor_in_val_fixture,
+    labor_in_histclim_fixture,
+    econvars_fixture,
+):
+    """
+    Test that calculate_labor_damages correctly concatenates damages across batches using multiprocessing
```

Comment:
This function doesn't seem to concatenate across batches. It seems to compare each batch separately. 

How does it test multiprocessing?

Reply:
The docstring here is wrong, will update it.

`calculate_labor_damages` func uses parallel processing. I'm not sure if parallel processing is tested in this way. Also, I agree calling it multiprocessing might be inappropriate.

Reply:
This is resolved by https://github.com/ClimateImpactLab/dscim/pull/68/commits/7cf92eb818d8d6a997a3f74a9852386dfa3c1074